package.json

```diff
@@ -33,7 +33,7 @@
     "init:generated": "mkdirp ./generated/loaders && mkdirp ./generated/plugins ",
     "lint": "run-s lint:*",
     "lint:js": "eslint . --ext .js,.jsx,.md",
-    "lint:markdown": "markdownlint --config ./.markdownlint.json *.md ./src/content/**/*.md",
+    "lint:markdown": "markdownlint --rules markdownlint-rule-emphasis-style --config ./.markdownlint.json *.md ./src/content/*.md ./src/content/**/*.md",
```

Comment:
It this rules necessary, Shouldn't it get it from the config file?

Reply:
hmm good question, also i just noticed: `*.md ./src/content/*.md` addition, is this necessary? @byzyk

Reply:
@montogeek please take a look at the PR description, this rule is not the part of `markdownlint` core package yet, I've created a custom rule for this which later can be merged into core if it proves itself valuable. See discussion in DavidAnson/markdownlint#150

@EugeneHlushko I've added `./src/content/*.md` since MD files placed in `content` root (e.g. `content/branding.md`) folder were not covered by existing scope.

Reply:
@montogeek forgot to mention that the actual rule configuration comes from config file. We need `--rule` here just to extend core package with custom rules.

Reply:
I mean shouldn't `*.md` cover all `.md` files

Reply:
For some reason, files in `content` root folder were not covered. We can attempt to address this in another PR since it's not really related to this one.

The way I discovered this is when `lint-staged` reported some MD linter issues on those files while `npm run lint:markdown` didn't catch those, that's why I explicitly added the root folder for now.

Reply:
hmm i am fine tackling this in other pr since this already does what it needs to, what do you think @montogeek ?

Reply:
It is fine